books/free-programming-books-pt_BR.md

```diff
@@ -83,6 +84,13 @@
 * [Google Android: Uma abordagem prática e didática](https://leanpub.com/google-android) - Rafael Guimarães Sakurai - (Necessário criar uma conta (gratuita) no Leanpub para baixar o livro completo nos formatos PDF, EPUB, MOBI ou pelo próprio site)
 
 
+### Arduino
+
+* [30 Projetos com Arduino](https://www.academia.edu/36112333/30_Projetos_com_Arduino_pdf) - Simon Monk (É necessario uma conta na Academia.edu para baixar o PDF)
```

Comment:
We will accept links to Academia.edu only if they have been posted there by the author. So we won't link to this one.